ui-participant/src/index.scss

```diff
@@ -54,6 +54,7 @@ code {
 
 a {
   text-decoration: none;
+  color: var(--brand-color);
```

Comment:
It might be worth adding a fallback here or scoping this `.App a` as the brand color variables are only defined within the App container.

https://github.com/broadinstitute/juniper/blob/fdb7d4267740b90906ea9e611b8d9f2b18b67359/ui-participant/src/App.tsx#L105-L108

Though practically, there might not be any links outside of the App container.

Reply:
that makes sense, thanks!

Reply:
adding .App breaks some button styling, so I added a default